src/argus/htmx/incident/filter.py

```diff
@@ -249,10 +259,9 @@ def _convert_filterblob(filterblob):
     return filterblob
 
 
-def _normalize_form_data(request):
+def _normalize_form_data(raw_data):
```

Comment:
While we're at it, we can even add a type hint:
```suggestion
def _normalize_form_data(raw_data: QueryDict):
```